modules/openid_connect/spec/lib/open_project/openid_connect/hooks/hook_spec.rb

```diff
@@ -66,6 +66,26 @@
         expect(session).to be_empty
       end
     end
+
+    context "when expires_in is missing" do
+      let(:expires_in) { nil }
+
+      it "does not store an expires_in" do
+        call_hook
+
+        expect(session["omniauth.oidc_expires_in"]).to be_nil
+      end
+    end
+
+    context "when expires_in is passed as a string" do
+      let(:expires_in) { "7200" }
+
+      it "does not store an expires_in" do
```

Comment:
This description seems to be inaccurate! @NobodysNightmare

Reply:
Well spotted. Apparently the test was written a bit too fast xD